lib/src/model/model_element.dart

```diff
@@ -1097,7 +1097,7 @@ abstract class ModelElement extends Canonicalization
         }
         if (params == null && element is FunctionTypedElement) {
           if (_originalMember != null) {
-            params = (_originalMember as dynamic).parameters;
```

Comment:
this is a good one to get rid of.  That might cover up some nasty bugs otherwise.  Don't remember why that is there...

Reply:
Haha it was a little weird.